shibata added English to some eadme.md and translated vantiq-install-closed-network-aws.md and fluentd.md (#24)
fujitake/vantiq-related: vantiq-apps-development/1-day-workshop/docs/eng/5-02_AppBuilder.md

## SplitByGroup

* Split the stream by groups.
* Used prior to activities that need to be handled by a specific group (not by event), such as `Dwell`, `ComputeStatistics`.

> * By splitting the stream, the resources used are distributed, and therefore load balancing is achieved.

Reviewer: ```suggestion

```

## ComputeStatistics

* Compute statistics for a single property contained of events that pass through the task.
* The statistical processing is done in memory, and as the output of the task, the input events are output as is.
* To get the statistics, use the Procedure for accessing the automatically generated statistics.
* Statistics items are the number of events, minimum, maximum, median, mean, and standard deviation.

![ComputeStatistics](../../imgs/02_AppBuilder/computestatistics_01.png)

① Output of the previous task which will be the input.  
```
{
   "TempSensorID": "tempSensor2",
   "Time": "2021-10-04T06:48:19.218Z",
   "Temp": 211,
   "PumpID": "pumpId2"
}
```

② Output of `ComputeStatistics` (the same as in ①).
```
{
   "TempSensorID": "tempSensor2",
   "Time": "2021-10-04T06:48:19.218Z",
   "Temp": 211,
   "PumpID": "pumpId2"
}
```

③ Procedure for access to statistics.  

* Procedures called <task name>StateGet, <task name>StateReset, and <task name>StateGetUpdate are generated automatically.
* To get the statistics, run <task name>StateGet from the VAIL code (in Procedure, Rule, Transform task, etc.) which is in any location.

> * If using "SplitByGroup" beforehand, the statistics will be kept for each partition, so "partitionKey" is required as an argument at runtime, and the value which was entered in the "groupBy" property of SplitByGroup should be entered for this value.  

Reviewer: ```suggestion
* If using "SplitByGroup" beforehand, the statistics will be computed for each partition, so "partitionKey" is required as an argument at runtime, and the value which was entered in the "groupBy" property of SplitByGroup should be used for that.  
```

> * If not using "SplitByGroup", it will be global.

Reviewer: ```suggestion
* If not using "SplitByGroup", it will be treated as global.
```

}
```

## VAIL

> * VAIL allows to describe the process freely.  

Reviewer: ```suggestion
* VAIL allows to run an arbitrary snippet of VAIL code.  
```
